include/irr/asset/CShaderIntrospector.h

```diff
@@ -67,20 +67,25 @@ class CIntrospectionData : public core::IReferenceCounted
 class CShaderIntrospector : public core::Uncopyable
 {
 public:
-    using SEntryPointStagePair = std::pair<std::string, E_SHADER_STAGE>;
+    struct SEntryPoint_Stage_Extensions
+    {
+        std::string entryPoint;
+        E_SHADER_STAGE stage;
+        core::SRange<const std::string> GLSLextensions;
```

Comment:
I'd use a smart refctd array, this could crash badly if I kill the driver (close device)